utility_code/lib_py3/item_replacement_substitutions.py

```diff
@@ -79,7 +79,10 @@ def _init_unnamed_items(cls):
             r'''{Items:[{Count:1b,Slot:0b,id:"minecraft:potion",tag:{Potion:"minecraft:strong_turtle_master",display:{Name:'{"text":"Potion of the Turtle Master"}'},plain:{display:{Name:"Potion of the Turtle Master"}}}}]}''',
         )
         named_chests = (
-            r'''{Items:[{Count:1b,Slot:13b,id:"minecraft:bread",tag:{display:{Name:'{"extra":[{"bold":false,"italic":false,"underlined":false,"strikethrough":false,"obfuscated":false,"color":"white","text":"Pretzel"}],"text":""}'},plain:{display:{Name:"Pretzel"}}}}]}''',
+            # Second try on Saturation Suspicious Stew -> Dichen Specialty Stew
+            r'''{Items:[{Count:1b,Slot:0b,id:"minecraft:suspicious_stew",tag:{Effects:[{EffectDuration:8,EffectId:23b}]}}]}''',
```

Comment:
I'm pretty sure the code for this list doesn't handle items with no name. You'll want to add the name tag to this item, and move it to the unnamed_chests list above. I really need to add comments about this for future reference.
```py
# These two lists allow replacing items that are missing lore text and/or item names, which would otherwise be ignored. The item NBT must match exactly with the exception of adding the name of the item for replacement purposes, including the order of data in lists, and the exact raw json text strings (formatting has the same capitalization, order, spaces, yada yada)

# These items do not have display names in-game; the only difference from the items to be replaced should be the addition of the display name for the sake of replacements. If the NBT otherwise matches exactly, the name and lore text are set on the items before continuing with item updates.
unnamed_chests = (

# ...

# These items have no lore text, which means they're assumed to be on mobs and will be skipped to prevent them from dropping or changing stats. Items that match exactly will be given a temporary line of lore so they can be replaced.
named_chests = (
```